extensions/amp-list/0.1/amp-list.js

```diff
@@ -1016,7 +1016,7 @@ export class AmpList extends AMP.BaseElement {
       return this.maybeResizeListToFitItems_();
     };
 
-    if (this.isLayoutContainer_) {
+    if (!this.loadMoreEnabled_ & this.isLayoutContainer_) {
```

Comment:
```suggestion
    if (!this.loadMoreEnabled_ && this.isLayoutContainer_) {
```

Reply:
My pre-coffee brain is showing 😅 Thanks to you and @choumx both for catching this